src/spyglass/spikesorting/spikesorting_sorting.py

```diff
@@ -203,9 +203,12 @@ def make(self, key: dict):
             )
         else:
             # whiten recording here if indicated, so can make sure dtype is float16
-            if sorter_params["whiten"]:
-                recording = sip.whiten(recording, dtype="float16")
-                sorter_params["whiten"] = False  # set whiten to False
+            if "whiten" in sorter_params.keys():
+                if sorter_params["whiten"]:
+                    sorter_params["whiten"] = False  # set whiten to False
+            # whiten recording separately; make sure dtype is float32
+            # to avoid downstream error with svm
+            recording = sip.whiten(recording, dtype="float32")
```

Comment:
```suggestion
            recording = sip.whiten(recording, dtype=np.float32)
```

Reply:
I'll make this change, and wanted to just point out that the function spikeinterface employs does this:
``` python
dtype = np.dtype(dtype)
```
So this change would only impact the readability of our own code.